packages/react-core/src/components/Tabs/Tab.tsx

```diff
@@ -49,6 +53,7 @@ const TabBase: React.FunctionComponent<TabProps> = ({
   href,
   innerRef,
   tooltip,
+  closeAriaLabel,
```

Comment:
Should this have a default or warning if not set since we render the icon (meaning it will always be an icon - no text - I'm assuming)? Same for the add button `aria-label`. Not sure if that's a best practice or not.